spec/calculate_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'calculate'
+describe Calculate_order do
+
+  let (:including_class){ Class.new { include Calculate_order} }
+
+  test_menu = {:Hamburger => 4.50, :Chips => 1}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)